app/controllers/application_controller.rb

```diff
@@ -300,6 +300,32 @@ def set_media
     @menu_media = Media.by_name - [Media.find_by_name(ArchiveConfig.MEDIA_NO_TAG_NAME), uncategorized] + [uncategorized]
   end
 
+  ### GLOBALIZATION ###
+
+#  before_filter :load_locales
+#  before_filter :set_preferred_locale
+
+#  I18n.backend = I18nDB::Backend::DBBased.new
+#  I18n.record_missing_keys = false # if you want to record missing translations
+  protected
+
+  def load_locales
+    @loaded_locales ||= Locale.order(:iso)
+  end
+
+  # Sets the locale
+  def set_preferred_locale
+    # Loading the current locale
+    if session[:locale] && @loaded_locales.detect { |loc| loc.iso == session[:locale]}
+      set_locale session[:locale].to_sym
```

Comment:
That sounds likely as that code was removed in commit a42de43baf7337ba71b47b72516b286f39bf48e6 (PR 2834) a couple of weeks ago.